### ISSUE TITLE
Exception with machinename target on asp.net core

### ISSUE THREAD

**Issue by Sebosek** [NLog/NLog#1374]
Hi,

`${machinename}` not supported in asp.net core.

**Type**:
- Bug

**NLog version**: 4.4.0-beta4

**Platform**: asp.net core (moniker dotnet5.4)

**Current NLog config** (xml or C#, if relevant)

```
        public static LoggingConfiguration CreateConfiguration(IFileLogConfiguration configuration)
        {
            var config = new LoggingConfiguration();
            var target = new FileTarget
            {
                FileName = configuration.FileName, Encoding = Encoding.UTF8, Layout = string.Join(LogSeparator, Columns)
            };
            var rule = new LoggingRule("*", NLog.LogLevel.Trace, target);

            config.AddTarget("file", target);
            config.LoggingRules.Add(rule);

            return config;
        }

        private static IEnumerable<string> Columns
        {
            get
            {
                yield return "${level}";
                yield return "${longdate}";
                yield return "${exception:format=Type}";
                yield return "${exception:format=Method}";
                //yield return "${asp-request:serverVariable=HTTP_METHOD}"; // Server variables aren't supported in DNX
                yield return "${machinename}";
                yield return "${document-uri}";
                yield return "${identity:name}";
                yield return "${message}";
            }
        }
```

In case of a BUG:

- What is the current result? ArgumentException
- What is the expected result? Current machine name
- Did you checked the [Internal log](https://github.com/NLog/NLog/wiki/Internal-Logging)? Well, application throws exception at startup...
- Please post full exception details (message, stacktrace, inner exceptions)
System.ArgumentException was unhandled by user code
  HResult=-2147024809
  Message=LayoutRenderer cannot be found: 'machinename'
  Source=NLog
  StackTrace:
       at NLog.Config.Factory`2.CreateInstance(String name)
       at NLog.Layouts.LayoutParser.ParseLayoutRenderer(ConfigurationItemFactory configurationItemFactory, SimpleStringReader sr)
       at NLog.Layouts.LayoutParser.CompileLayout(ConfigurationItemFactory configurationItemFactory, SimpleStringReader sr, Boolean isNested, String& text)
       at NLog.Layouts.SimpleLayout.set_Text(String value)
       at NLog.Layouts.SimpleLayout..ctor(String txt, ConfigurationItemFactory configurationItemFactory)
       at NLog.Layouts.Layout.FromString(String layoutText, ConfigurationItemFactory configurationItemFactory)
       at NLog.Layouts.Layout.FromString(String layoutText)
       at NLog.Layouts.Layout.op_Implicit(String text)
       at ServiceHub.Logger.LoggerConfigurator.CreateConfiguration(IFileLogConfiguration configuration) in C:\Logger\Logger\src\Logger\LoggerConfigurator.cs:line 18
       at ServiceHub.Logger.Logger..ctor(IFileLogConfiguration configuration) in C:\Logger\Logger\src\Logger\Logger.cs:line 17
       at ServiceHub.Logger.Provider.NLogProvider.CreateLogger(String name) in C:\Logger\Logger\src\Logger\Provider\NLogProvider.cs:line 29
       at Microsoft.Extensions.Logging.Logger.AddProvider(ILoggerProvider provider)
       at Microsoft.Extensions.Logging.LoggerFactory.AddProvider(ILoggerProvider provider)
       at ServiceHub.Logger.Extensions.LoggerFactoryExtensions.AddServiceHubLogger(ILoggerFactory factory, IFileLogConfiguration configuration) in C:\Logger\Logger\src\Logger\Extensions\LoggerFactoryExtensions.cs:line 11
       at ServiceHub.Logger.Sample.Startup.Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory) in C:\Logger\Logger\src\Logger.Sample\Startup.cs:line 38
  InnerException: 
- Are there any work arrounds? yes/no no
- Is there a version in which it did worked? I don't know
- Can you help us by writing an unit test? yes

**Comment by 304NotModified** [comment]
Not sure if this is a bug, but we do need to fix some docs for coreclr support

**Comment by Sebosek** [comment]
Thank you for quick response; to be honest I'm not sure, what you mean as "we do need to fix some docs for coreclr support"..? 
However, I wrote our target renderer, but I'll ask my employee to permission send pull request, where I fixed machine name for .net core RC1...

**Comment by 304NotModified** [comment]
This is fixed in netstandard 1.5, See https://github.com/NLog/NLog/pull/1526
